_monitoring-plugins/ad/index.md

```diff
@@ -106,25 +125,26 @@ If you see the detector pending in "initialization" for longer than a day, aggre
 
 ![Anomaly detection results]({{site.url}}{{site.baseurl}}/images/ad.png)
```

Comment:
The screen capture is not latest. Either remove this old screen capture or replace it with new one.